src/libraries/Microsoft.PowerFx.Connectors/OpenApiParser.cs

```diff
@@ -131,6 +131,7 @@ internal class OpenApiParser
         private static bool IsSafeHttpMethod(HttpMethod httpMethod)
         {
             // HTTP/1.1 spec states that only GET and HEAD requests are 'safe' by default.
+            // https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html
             return httpMethod == HttpMethod.Get ||
```

Comment:
2616 is a bit old
https://www.rfc-editor.org/rfc/rfc7231#section-4.2.1
   Of the request methods defined by this specification, the GET, HEAD,
   OPTIONS, and TRACE methods are defined to be safe.

Reply:
Good points - I wonder if PowerApps should be updated. will followup separaretly...